.github/workflows/upgrade.yml

```diff
@@ -26,16 +26,16 @@ jobs:
           ./gradlew run --args='..'
 
       - name: Generate Github Token
-        uses: tibdex/github-app-token@v2
+        uses: actions/create-github-app-token@v2
         id: generate_token
         with:
-          app_id: ${{ secrets.APP_ID }}
-          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.APP_PRIVATE_KEY }}
 
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v7
         with:
-          branch-token: ${{ steps.generate_token.outputs.token }}
+          token: ${{ steps.generate_token.outputs.token }}
```

Comment:
This is required for workflows. See https://github.com/peter-evans/create-pull-request/blob/e5a03b409aae444ebc2a01147c555f89034030b6/action.yml#L4-L6